tools/generator-terraform/generator/mappings/assignment_direct.go

```diff
@@ -69,6 +69,10 @@ func (d directAssignmentLine) assignmentForCreateUpdateMapping(mapping resourcem
 		return d.schemaToSdkMappingBetweenListFields(mapping, schemaField, sdkField, sdkConstant, apiResourcePackageName)
 	}
 
+	if schemaField.ObjectDefinition.Type == resourcemanager.TerraformSchemaFieldTypeDictionary && sdkField.ObjectDefinition.Type == resourcemanager.DictionaryApiObjectDefinitionType {
+		return d.schemaToSdkMappingBetweenDictionaryOfSimpleTypes(mapping, schemaField, sdkField, sdkConstant, apiResourcePackageName)
```

Comment:
can we also add this to the Read function too, so that we can map these back the other way?